bigbluebutton-html5/imports/ui/components/video-provider/container.tsx

```
@@ -147,30 +149,41 @@ const VideoProviderContainer: React.FC<VideoProviderContainerProps> = (props) =>
+    stopVideo,
+    applyCameraProfile,
+    myRole,
+  };
+
+  switch (currentMeeting?.cameraBridge) {
+    case 'livekit':
+      return (
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        <LiveKitCameraBridge {...providerProps} />
+      );
+    case 'bbb-webrtc-sfu':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)